src/server.tsx

```diff
@@ -448,16 +448,20 @@ app.get(
             const countryCode = req.header('X-GU-GeoIP-Country-Code');
             const ampVariantAssignments = getAmpVariantAssignments(req);
             const epic = await ampEpic(ampVariantAssignments, countryCode);
+            const campaignCode = `AMP__${epic.testName}__${epic.variantName}`;
```

Comment:
Let's share this with the other [uses](https://github.com/guardian/support-dotcom-components/blob/45e59e05799443776434f260c56a745a864ce438/src/tests/amp/ampEpicTests.ts#L100) - we have a similar function for non-AMP here - https://github.com/guardian/support-dotcom-components/blob/45e59e05799443776434f260c56a745a864ce438/src/lib/tracking.ts

Reply:
Good shout @tomrf1, I'll do that now.